Documentation/linux-setup.md

```diff
@@ -21,34 +21,34 @@ Open a command prompt and run the commands below that match your distro:
 
 ```bash
 # Debian 8 - .NET Core 2.0 and newer
-wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg
+wget -nvO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg
```

Comment:
This gives me error:
```
furt@Ubuntu:~$ wget -nvO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor
wget: illegal option -- `-nO'
```
It should probably be 
`wget -nv -O -  https://packages.microsoft.com/keys/microsoft.asc`

Reply:
Ouch, I was too enthusiastic and did not check every command :blush: . Thx for catching it. 
Separating the switches (like you suggested) works now.